src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public interface OperationHandler {
+    List<FruitTransaction> getFruitFinalQuantity(List<FruitTransaction> list,
```

Comment:
The method 'getFruitFinalQuantity' should have a single responsibility. Having two lists as parameters might indicate that this method is doing more than one thing. Consider refactoring to ensure that each method has a single responsibility and that the interface adheres to SOLID principles.